src/group/group.controller.ts

```diff
@@ -45,6 +47,8 @@ export class GroupController {
     return this.groupService.create(createGroupDto, user.id);
   }
 
+  @Permissions(UserPermission.ViewGroups)
```

Comment:
We may have problems as well. This is Public endpoint, and if user is not authorised we'll show the page, though guard may prevent it for authorised user. 🤔